openshift-tools/pkg/discover-etcd-initial-cluster/initial-cluster.go

```diff
@@ -105,173 +113,193 @@ func (o *DiscoverEtcdInitialClusterOptions) Validate() error {
 	if len(o.TargetName) == 0 {
 		return fmt.Errorf("missing --target-name")
 	}
+	if len(o.TargetPeerURLPort) == 0 {
+		fmt.Errorf("missing TargetPeerURLPort")
+	}
+	if len(o.TargetPeerURLScheme) == 0 {
+		fmt.Errorf("missing TargetPeerURLScheme")
+	}
```

Comment:
Why aren't these optional? Do we expect these to be anything other than https and 2380?

Reply:
it is to accommodate testing we set defaults.

Reply:
testing uses `unixs` and nonstandard ports.

Reply:
OK. Thanks.

Reply:
> nit: no binding was added for these new flags in the BindFlags function above.

These are not configurable at runtime only used for testing.